app-template/copay/sass/app.scss

```diff
@@ -0,0 +1,754 @@
+.button.button-positive {
+  background-color: $primary-button-color;
+  color: $primary-button-text-color !important;
+}
+
+.button.button-primary.activated,
+.onboarding .button.button-primary.activated,
+#agree-to-terms .button.button-primary.activated,
+.button.button-positive.activated {
+  background-color: $primary-button-color;
+  opacity: 0.7;
+}
+
+/* Secondary */
+.button.button-secondary,
+.onboarding .button.button-secondary {
+  background-color: $secondary-button-color;
+  color: $secondary-button-text-color !important;
+}
+
+.button.button-secondary.activated,
+.onboarding .button.button-secondary.activated {
+  background-color: $secondary-button-color;
+  opacity: 0.7
+}
+
+/* Clear */
+.button.button-clear,
+.onboarding .button.button-clear {
+  color: $primary-button-color !important;
+}
+
+.button.button-clear.activated,
+.onboarding .button.button-clear.activated {
+  color: $primary-button-color;
+  opacity: 0.7;
+}
+
+/*
+ * Toggle buttons
+ */
+
+.toggle.toggle-balanced,
+.toggle.toggle-positive {
+  input:checked + .track {
+    border-color: $toggle-button-color;
+    background-color: $toggle-button-color;
+  }
+}
+
+/*
+ * Check buttons
+ */
+
+.checkbox-positive,
+.checkbox-balanced {
+  input:checked:before,
+  input:checked + .checkbox-icon:before {
+    background: $check-button-color;
+    border-color: $check-button-color;
+  }
+
+  input:before,
+  .checkbox-icon:before {
+    border-color: $check-button-color;
+  }
+}
+
+.checkbox input:checked:before,
+.checkbox input:checked + .checkbox-icon:before {
+  background: $check-button-color;
+  border-color: $check-button-color;
+}
+
+/*
+ * Popup modals
+ */
+
+.popup-modal-header-success {
+  background: $success-color;
+}
+
+.popup-modal-header-warning {
+  background: $alert-color;
+}
+
+.popup-modal-content-warning {
+  .button {
+    color: $alert-color !important;
+  }
+}
+
+/***********************************************************************************************************
+ *
+ * View specific styles
+ * 
+ ***********************************************************************************************************/
+
+/* 
+ * Starting view
+ */
+
+#starting {
+  background: $gradient-bg-top-color;
+  background: -moz-linear-gradient(top, $gradient-bg-top-color 0%, $gradient-bg-bottom-color 100%);
+  background: -webkit-gradient(left top, left bottom, color-stop(0%, $gradient-bg-bottom-color), color-stop(100%, $gradient-bg-bottom-color));
+  background: -webkit-linear-gradient(top, $gradient-bg-top-color 0%, $gradient-bg-bottom-color 100%);
+  background: -o-linear-gradient(top, $gradient-bg-top-color 0%, $gradient-bg-bottom-color 100%);
+  background: -ms-linear-gradient(top, $gradient-bg-top-color 0%, $gradient-bg-bottom-color 100%);
+  background: linear-gradient(to bottom, $gradient-bg-top-color 0%, $gradient-bg-bottom-color 100%);
+  color: #fff;
+}
+
+/*
+ * Onboarding views
+ */
+
+.onboarding {
+  color: $text-light-color !important;
```

Comment:
@ajp8164 this file includes a lot of overrides where none are needed – all onboarding styles are already defined here: https://github.com/bitpay/copay/blob/master/src/sass/views/onboarding/onboarding.scss

Can we just use variables to override these things? If we write everything twice, our scss will become very brittle.